### PR TITLE
Feat: monorepo

### PR DESCRIPTION
**Pull request by mauriciomutte**

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "packages": ["packages/*"],
+  "version": "0.0.1",
+  "npmClient": "yarn",
+  "useWorkspaces": true
+}
```
Comment (jaburcodes) [inline lerna.json]: We probably don't need lerna at all. Yarn workspaces is enough.

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    "type": "git",
+    "url": "https://github.com/wendelfreitas/animavita"
+  },
+  "author": "Wendel Freitas <wendelfreitasbranco@gmail.com>",
+  "bugs": {
+    "url": "https://github.com/wendelfreitas/animavita/issues"
+  },
+  "workspaces": {
+    "packages": [
+      "packages/*"
```
Comment (jaburcodes) [inline package.json]: nice

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    ]
+  },
+  "scripts": {
+    "app:start": "yarn workspace @animavita/mobile start",
+    "server:start": "yarn workspace @animavita/server start"
+  },
```
Comment (jaburcodes) [inline package.json]: can we add more commands, like, building the android/ios?

**Comment by jaburcodes** [comment]
Is react-native working? It prob needs some work arounds and rn doesn't support monorepo out of the box.

**Comment by wendelfreitas** [comment]
> Is react-native working? It prob needs some work arounds and rn doesn't support monorepo out of the box.

this weekend, I will test this

thanks for pr @mauriciomutte

**Comment by wendelfreitas** [comment]
@mauriciomutte you can put some instructions here for execution when trying to execute I get **No such file or directory: 'workspaces' on terminal**

thanks

**Comment by renanmav** [comment]
Thanks for all the work @mauriciomutte, it was amazing. I added a few things in #21, so I’ll close this in order to merge all changes.

Congrats 🥇

**Pull request merged**